tests/apps/stateapp/app.go

```diff
@@ -438,18 +520,27 @@ func grpcHandler(w http.ResponseWriter, r *http.Request) {
 		}
 		res.States = states
 	case "get":
+		if req, err = parseRequestBody(w, r); err != nil {
+			return
```

Comment:
same here.

Reply:
the error is handled inside `parseRequestBody`